src/app/tabs/tabs.page.html

```diff
@@ -12,17 +15,20 @@
     </ion-tab-button>
 
     <ion-tab-button tab="events" href="/events">
-      <ion-icon aria-hidden="true" name="flag-outline"></ion-icon>
+      <ion-icon aria-hidden="true" name="flag"></ion-icon>
       <ion-label>Събития</ion-label>
     </ion-tab-button>
 
     <ion-tab-button tab="notifications" href="/notifications">
-      <ion-icon aria-hidden="true" name="heart-outline"></ion-icon>
+      <ion-icon aria-hidden="true" name="heart"></ion-icon>
       <ion-label>Любими</ion-label>
     </ion-tab-button>
 
     <ion-tab-button tab="profile" href="/profile">
-      <ion-icon aria-hidden="true" name="person"></ion-icon>
+      <!-- <ion-icon aria-hidden="true" name="person"></ion-icon> -->
+      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```

Comment:
Same here - is this working when highlighting active tab?

Reply:
![Screenshot 2023-11-13 111321](https://github.com/Getsov/Motorsport-Events-App/assets/114027141/017d4b00-b461-4097-aaa9-f0c4b25fd9ac)

Reply:
The same here